packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceHeader.css

```diff
@@ -14,47 +14,59 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.TraecDiffHeader--traceHeader {
+.TraceDiffHeader--traceHeader {
   align-items: stretch;
-  background: #fafafa;
-  border-right: 1px solid #d8d8d8;
+  background: #fff;
   display: flex;
   flex-direction: column;
   flex: 1;
+  min-width: 0;
 }
 
-.TraecDiffHeader--traceTitle {
+.TraceDiffHeader--traceTitle {
   align-items: center;
-  background: #fff;
-  border-bottom: 1px solid #eee;
   display: flex;
   flex: 1;
-  font-size: 1.8em;
+  font-size: 1.7em;
   margin: 0;
-  padding: 0.25rem 2.5rem 0.25rem 1.25rem;
+  padding: 12px 40px 12px 15px;
```

Comment:
why are we changing to absolute pixels? It's not fluid-layout friendly. Jaeger UI typically works reasonably well at different zoom levels.